docs/scos/dev/migration-program/migration-to-paas/paas-migration-documents/install-the-latest-version-of-docker-sdk.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Install the latest version of Docker SDK
+description: This document describes how to install the latest version of Docker SDK.
+template: howto-guide-template
+---
+
+# Install the latest version of Docker SDK
+
+{% info_block infoBox %}
+
+## Resources for assessment Backend
+
+{% endinfo_block %}
+
+## If Docker SDK is not installed
+
+1. Follow the [installation guide](/docs/scos/dev/setup/installing-spryker-with-docker/docker-installation-prerequisites/installing-docker-prerequisites-on-macos.html#system-requirements-for-installing-spryker-with-docker).
```

Comment:
the content of the linked doc doesn't match the description. Should I install only the prerequisites?